### DISCUSSION TITLE
Specity length ratio

### DISCUSSION THREAD

**Discussion by ghost** [gyscos/cursive#644] [Q&A]
```rust
let layout = LinearLayout::horizontal()
        .child(first_panel)
        .child(second_panel);
```

I want them to fill all terminal window and using `add_fullscreen_layer` for that.
how can I specify that I want `first_panel` to fill 30% and the second panel to cover the rest 70%?

**Comment by gyscos** [comment]
Hi, and thanks for the question!

Right now `LinearLayout` does not support specifying sizes in percentage like that.

Workarounds include using a `OnLayoutView<FixedLayout>` and updating the sizes yourself during the `layout` call.
